Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -72,32 +79,32 @@ Cilium replacement has been installed:
 
 .. include:: k8s-install-download-release.rst
 
-Next, generate the required YAML files and deploy them. **Important:** Replace
-``REPLACE_WITH_API_SERVER_IP`` and ``REPLACE_WITH_API_SERVER_PORT`` below with the concrete
-control-plane node IP address and the kube-apiserver port number reported by ``kubeadm init``
-(usually, it is port ``6443``).
+Next, generate the required YAML files and deploy them. **Important:** Make sure you correctly set your
+``API_SERVER_IP`` and ``API_SERVER_PORT`` below with the control-plane node IP address and the kube-apiserver port number reported by ``kubeadm init`` (kubeadm will use port ``6443`` by default).
```

Comment:
`kubeadm` -> `Kubeadm`? Since you fixed it above :)

Reply:
thanks yup fixed this one and one on line 75